scripts/cert-update.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+eval `ssh-agent`
```

Comment:
this will turn on the ssh agent?

https://explainshell.com/explain?cmd=eval+%60ssh-agent%60

interesting.